website/docs/introduction/push-notifications.md

```diff
@@ -0,0 +1,47 @@
+---
+id: push-notifications
+title: Push Notifications
+---
+
+A push notification _is a message that is "pushed" from backend server or application to user interface, e.g. 
+(but not limited to) mobile applications and desktop applications_(see [wikipedia](https://en.wikipedia.org/wiki/Push_technology#Push_notification)).
+
+Push notification main advantage is to keep user interest high and avoid your application getting forgotten and uninstalled 
```

Comment:
How about something like "Push notifications allow your services to signal to you application there there is work it needs to do.  This can include background work like synchronizing data or user visible work like displaying relevant.  Push messages can be used to keep your application fresh and relevant so it isn't forgotten and uninstalled.